DNN Platform/Library/Obsolete/HostController.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Entities.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using DotNetNuke.Common;
+    using DotNetNuke.ComponentModel;
+
+    using Microsoft.Extensions.DependencyInjection;
+
+    using INewHostController = DotNetNuke.Abstractions.Entities.Controllers.IHostController;
+
+    // None of the APIs are deprecated, but the IHostController
+    // is deprecated and moved to the abstractions project. When
+    // it is time to remove APIs we should remove the parent
+    // classes listed here
+    /// <inheritdoc />
+    public partial class HostController : ComponentBase<IHostController, HostController>, IHostController
+    {
+        private static IHostController currentInstance;
+
+        [Obsolete("Deprecated in 9.7.1. Scheduled for removal in v11.0.0, use DotNetNuke.Abstractions.Controllers.IHostController instead.")]
```

Comment:
It would be great if we could give more of of the "how" to fix so that people could know more than just what interface to look at.

Reply:
I agree, but I am not sure what to put down. The Dependency Injection story is slightly different depending on what part of the platform you are working in. Should it be as simple as this?

```suggestion
        [Obsolete("Deprecated in 9.7.1. Scheduled for removal in v11.0.0, use DotNetNuke.Abstractions.Controllers.IHostController via Dependency Injection instead.")]
```

Reply:
I might say _resolve_ instead of _use_, but I don't think we can add more details than _via Dependency Injection_, unless we add a link (to dnndocs?)

Reply:
The verbiage has been updated for the deprecation messages to

```
[Obsolete("Deprecated in 9.7.1. Scheduled for removal in v11.0.0, use DotNetNuke.Abstractions.IHostSettingsService instead.")]
```